nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/log/kafka/KafkaLogTailer.java

```diff
@@ -343,6 +355,20 @@ public void commit() {
         }
     }
 
+    /**
+     * Commits the consumer at its current position regardless of lastOffsets or lastCommittedOffsets
+     */
+    protected void forceCommit() {
+        log.info("Force commit after a move");
+        final Map<TopicPartition, OffsetAndMetadata> offsets = new HashMap<>(topicPartitions.size());
```

Comment:
We don't use `final` for local variables (noise for the reader). It's not needed for the lambda because the compiler can detect that it's _effectively final_.